src/Microsoft.ML.StandardLearners/Standard/Simple/SimpleTrainers.cs

```diff
@@ -65,11 +63,35 @@ public RandomTrainer(IHostEnvironment env, Arguments args)
             Host.CheckValue(args, nameof(args));
         }
 
+        public BinaryPredictionTransformer<RandomPredictor> Fit(IDataView input)
+        {
+            var cachedTrain = Info.WantCaching ? new CacheDataView(Host, input, prefetch: null) : input;
+
+            RoleMappedData trainRoles = new RoleMappedData(cachedTrain);
+            var pred = Train(new TrainContext(trainRoles));
+            return new BinaryPredictionTransformer<RandomPredictor>(Host, pred, cachedTrain.Schema, trainRoles.Schema.Feature.Name);
+        }
+
         public override RandomPredictor Train(TrainContext context)
         {
             Host.CheckValue(context, nameof(context));
             return new RandomPredictor(Host, Host.Rand.Next());
         }
+
+        /// <summary>
```

Comment:
>summary [](start = 13, length = 7)

this is not a good comment. 
A good comment has to describe the purpose of the method, and not its implementation.

In this particular case, I think, it's not necessary to have a summary comment at all.